spec/integration/categories_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
```

Comment:
There are errors when I try to run `rspec` or the commands from your README to run specs. Kindly see the screenshots below. 

![tests](https://user-images.githubusercontent.com/35504149/202527896-bed31528-f319-4988-a1a0-6e2233ec1e29.png)


![tests2](https://user-images.githubusercontent.com/35504149/202527924-0a63a839-0118-459f-80f8-087b69a4f0b9.png)

Reply:
### The tests are working perfectly fine on a Linux environment.


https://user-images.githubusercontent.com/17243325/202533572-4ecbede8-09e8-4507-a314-da336978f37b.mp4